.circleci/config.yml

```diff
@@ -13,8 +13,7 @@ jobs:
     steps:
       - checkout
       - run: yarn global add @percy/agent lerna@4.0.0
-      - run: yarn bootstrap
-      - run: npm rebuild node-sass
+      - run: yarn bootstrap      
```

Comment:
sass does not need to rebuilt like node-sass as it is a javascript implementation